src/org/labkey/test/tests/assay/UploadLargeExcelAssayTest.java

```diff
@@ -133,7 +135,9 @@ public void testUpload200kRows() throws Exception
         setFormElement(Locator.input("__primaryFile__"), largeExportExcelFile);
         clickButton("Save and Finish");
 
-        waitForPipelineJobsToComplete(2, "200k take 2", false, 12 * 60000);
+        var assayJobsPage2 = new AssayUploadJobsPage(getDriver());
+        var pipelineDetailsPage2 = assayJobsPage2.clickJobStatus("200k take 2", 3 * getDefaultWaitForPage());
```

Comment:
This multiplies by `getDefaultWaitForPage()` whereas line 116 multiplies by `WebDriverWrapper.WAIT_FOR_PAGE`. Are they the same, and is one superior to the other?

Reply:
Yes, they are the same, and no, I don't think one is any better than the other